src/SourceBuild/content/prep-source-build.sh

```diff
@@ -223,6 +223,7 @@ fi
 
 if [ "$removeBinaries" == true ]; then
 
+  extracted_tarball=false
```

Comment:
nit: better to use 0 and 1 instead of true and false. Unlike powershell which has $True and $False boolean vars, these true and false are the actual shell util commands. At least in new code we can try to avoid this anti-pattern.

Reply:
I originally used `true`/`false` to stay consistent with the rest of the script. Changed now to 0/1.